internal/progressPrinter/progress.go

```diff
@@ -103,7 +103,6 @@ func LogMessage(message string) {
 //
 // no need to instanciate, it is a singleton, only one instance already started before use.
 func AddTracker(key string, title string, total int64) string {
```

Comment:
Is this a change related to the issue? This sleeps is to give animation effect on the start of new bars?

Reply:
I never saw the animation to be honest. I would like to have it as quick as possible in the screen to speed up the job.

if you want to keep it, can we decrease the waiting time by half?